modules/xfeatures2d/src/logos.cpp

```diff
@@ -0,0 +1,80 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+/*
+ * MIT License
+ *
+ * Copyright (c) 2018 Stephanie Lowry
```

Comment:
Is this file code covered by 3rdparty license too? Is where some adopted code or it has been written from scratch ?

Reply:
I have removed the license.